GameData/ROCapsules/PartConfigs/GeminiBDB/Waterfall_Support_GeminiBDB.cfg

```diff
@@ -0,0 +1,134 @@
+@PART[ROC-GeminiEquipmentSectionBDB]:BEFORE[ROWaterfall]:NEEDS[Waterfall]
+{
+    ROWaterfall
+    {
+        template = rowaterfall-rcs-hypergolic-1
+        audio = rcs-jet-1
+        transform = rcsTransform
+        position = 0,0,0
+        rotation = 0, 0, 180
+        scale = 0.7, 0.7, 0.7
+    }
+}
+@PART[ROC-GeminiEquipmentSectionBDB]:AFTER[zROWaterfall_99_Finalize]:NEEDS[Waterfall]
+{
+    @EFFECTS
+    {
+        @running
+        {
+            |_ = rcsAtt
+        }
+    }
+}
+
+
+@PART[ROC-GeminiOAMS*BDB]:BEFORE[ROWaterfall]:NEEDS[Waterfall]
+{
+    ROWaterfall
+    {
+        template = waterfall-hypergolic-white-upper-1
+        audio = rcs-jet-1
+        transform = thrustTransform
+        position = 0,0.025,-0.015
+        rotation = 0, 0, 0
+        scale = 0.07, 0.07, 0.1
+
+        ExtraTemplate
+        {
+            template = rowaterfall-glow-hypergolic-white
+            position = 0,0.0065,0.0209
+            rotation = -26.1, 0, 0
+            scale = 0.12, 0.15, 0
+        }
+    }
+}
+@PART[ROC-GeminiOAMSRCSBDB]:AFTER[zROWaterfall_99_Finalize]:NEEDS[Waterfall]
+{
+    @MODULE[ModuleWaterfallFX]
+    {
+        @CONTROLLER[throttle]
+        {
+            @linkedTo = rcs
+            thrusterTransformName = thrustTransform
```

Comment:
Guarantee this field did not exist before?  Consider future-proofing with `%thrusterTransformName`.

Reply:
Yes, the key doesn't exist because we generated the controller ourselves.

I highly doubt that the throttle controller will ever accept a `thrusterTransformName`, because that key is specifically for RCS controllers to differentiate between multiple `ModuleRCSFX`es.